source/includes/entityref/_federaltaxform.md

```diff
@@ -0,0 +1,154 @@
+
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>FederalTaxForm fields</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="even">
+            <td>id</td>
+            <td>Integer</td>
+            <td>Unique identifier for this entity.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>candidate</td>
+            <td>To-one association</td>
+            <td>Candidate</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>customDate1-5</td>
+            <td>Timestamp</td>
+            <td>Configurable date fields that can be used to store custom data depending on the needs of a particular deployment.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="odd">
+            <td>customInt1-5</td>
+            <td>Integer</td>
+            <td>Configurable numeric fields that can be used to store custom data depending on the needs of a particular deployment.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>customMoney1-5</td>
+            <td>BigDecimal</td>
+            <td>Configurable numeric fields that can be used to store custom data depending on the needs of a particular deployment.</td>
+            <td></td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>customText1-10</td>
+            <td>String (100)</td>
+            <td>Configurable text fields that can be used to store custom data depending on the needs of a particular deployment.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>dateAdded</td>
+            <td>Timestamp</td>
+            <td>The date on which this record was created in the Bullhorn system.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>dateLastModified</td>
+            <td>Timestamp</td>
+            <td>The date on which this record was last modified.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>federalAdditionalWithholdingsAmount</td>
+            <td>BigDecimal</td>
+            <td>	Number of federal withholdings the Candidate has selected on his or her W-2 tax form.</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr class="odd">
+            <td>federalExemptions</td>
+            <td>Integer</td>
+            <td>Number of federal exemptions the Candidate has indicated on his or her W-2 tax form.</td>
```

Comment:
Favor gender-neutral language. Change to:
Number of federal withholdings selected on W-2 tax form.